src/action/linux/selinux/build.sh

```diff
@@ -0,0 +1,5 @@
+#! /usr/bin/env nix-shell
+#! nix-shell -i bash -p libselinux -p semodule-utils -p checkpolicy
```

Comment:
We should probably pin this using our flake rather than taking the packages from whatever the user's nixpkgs is set to.

And what are the chances we could build this as part of the build process rather than checking binary files into the repo?

Reply:
`nix-installer` is also a valid Rust tool, `cargo install nix-installer`. No way to do it. :(

Reply:
Right. RIP.